lib/validate.js

```diff
@@ -30,8 +30,8 @@ module.exports = {
     };
 
     const getEntryExtension = fileName => {
-      const files = glob.sync(`${fileName}.*`, {
-        cwd: this.serverless.config.servicePath,
+      const files = globby.sync(`${fileName}.*`, {
+        cwd: this.serverless.config.servicePath || process.cwd(),
```

Comment:
I'm not sure if defaulting to CWD is right here. It will miss the handlers in case CWD !== servicePath and render the webpack configuration invalid. In general, serverless-webpack depends on servicePath everywhere and uses it to create it's outputs.

Reply:
I added this since `servicePath` was null in some cases (maybe just in tests) and globby doesn't support that while glob did. I verified that both libs default to process.cwd if cwd is not set so this should be fine.

Reply:
Yes, that can be the tests. Not all of them intialize a full config object for testing. So this should be ok 👍

Reply:
BTW: I just remember another open PR, that stabilizes the directories and uses webpackconfig.context to keep the webpack root directory. So this line would be void if the two PR are merged together.
https://github.com/serverless-heaven/serverless-webpack/pull/325